virtual-desktop/src/app/window-manager/simple-window-manager/simple.component.ts

```diff
@@ -40,27 +41,35 @@ export class SimpleComponent implements OnInit {
     let requestedPluginID: string = (window as any)['GIZA_PLUGIN_TO_BE_LOADED'];
 
     if (!requestedPluginID) {
-      // Default back to sample plugin:
-      this.logger.warn("GIZA_PLUGIN_TO_BE_LOADED not set, defaulting to sample plugin!");
-      requestedPluginID = "com.rs.mvd.myplugin";
+      this.logger.severe("Plugin ID not given, cannot continue!");
```

Comment:
If you removed the requestedPluginID from being set to a default, and you say you cannot continue, shouldn't the rest of the logic not run? requestPluginID is being used farther down the method. Either way if not breaking things, it seems to attempt to do pointless things